authors/apps/comments/views.py

```diff
@@ -123,3 +123,57 @@ def delete(self, request, pk, article_id):
         return Response({
             "error": "You do not have permissions to delete this comment."
         }, status=status.HTTP_403_FORBIDDEN)
+
+
+class Like(APIView):
+    """
+    create and get likes.
+    """
+
+    permission_classes = (IsAuthenticated, )
+
+    def get_comment(self, comment_id):
+        """search for a comment by id."""
+        try:
+            return Comment.objects.get(id=comment_id)
+        except Comment.DoesNotExist:
+            raise NotFound({"error": "The comment does not exist."})
+
+    def get_like(self, comment_id, username):
+        try:
+            return Likes.objects.get(comment=comment_id,commenter_id=username)
+        except Likes.DoesNotExist:
+            return None
+
+    def get(self, request, **kwargs):
+        """Method to check for your like on a comment"""
+        like = self.get_like(kwargs['comment_id'], request.user.username)
+        if like:
+            return Response({"Message":"True, You have ever liked this comment."},
```

Comment:
This could be a better response.
```suggestion
            return Response({"status":true},
```